### PR TITLE
NOJIRA, CalNetLdap search returns berkeleyeduaffiliations as Enumerable

### PR DESCRIPTION
**Pull request by johncrossman**
Master PR is #5946 

The [Bamboo breakage we saw on master](https://bamboo.ets.berkeley.edu/bamboo/browse/MYB-CLCMVPMASTER-4601) is likely to impact QA branch, too.

### CODE REVIEW

File: spec/models/calnet_ldap/client_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   it 'should initialize with a configured Net::LDAP object' do
     ldap = subject.instance_variable_get :@ldap
-    expect(ldap).to be_an_instance_of Net::LDAP
+    expect(ldap).to be_a Net::LDAP
```
Comment (redconfetti) [inline spec/models/calnet_ldap/client_spec.rb]: What is `be_a` used for? The class constant itself rather than an instance of the class?

Comment (redconfetti) [inline spec/models/calnet_ldap/client_spec.rb]: Nevermind, I'm looking up. [type check matchers](https://www.relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/type-check-matchers)

I see that the kind_of check used by `be_a` sees if the object includes functionality inherited or mixed in from Net::LDAP, whereas `be_an_instance_of` performs an `#instance_of?` call that only returns true if the object is of a specific class.

File: spec/models/calnet_ldap/user_attributes_spec.rb [hunk 2]
```diff
@@ -8,12 +8,12 @@
     let(:ldap_result) do
       {
         dn: ['uid=61889,ou=people,dc=berkeley,dc=edu'],
-        objectclass: ['top', 'eduPerson', 'inetorgperson', 'berkeleyEduPerson', 'organizationalperson', 'person', 'ucEduPerson'],
+        objectclass: %w(top eduPerson inetorgperson berkeleyEduPerson organizationalperson person ucEduPerson),
```
Comment (redconfetti) [inline spec/models/calnet_ldap/user_attributes_spec.rb]: oh so ruby

File: spec/models/calnet_ldap/client_spec.rb (hunk 1 above)
Comment (johncrossman) [inline spec/models/calnet_ldap/client_spec.rb]: thanks, @redconfetti  You're right – in this case, we don't need the restrictiveness of `be_an_instance_of`.

**Pull request merged**